components/typography/styles.js

```diff
@@ -112,7 +115,7 @@ export const _generateBodySmallStyles = (selector) => {
 			color: var(--d2l-color-tungsten);
 			font-size: 0.7rem;
 			font-weight: 400;
-			line-height: 1rem;
+			line-height: ${unsafeCSS(inputStyleTweaksEnabled ? '0.9rem' : '1rem')}; /* stylelint-disable-line */
```

Comment:
Stylelint got confused by this JS-in-CSS and wanted double-quotes. Ignoring.

Reply:
I think I ran into this recently too!